core/src/main/scala/sttp/openai/OpenAi.scala

```diff
@@ -80,6 +67,103 @@ class OpenAi(authToken: String) {
       .body(chatBody)
       .response(asJsonSnake[ChatResponse])
 
+  /** Fetches all files that belong to the user's organization from [[https://platform.openai.com/docs/api-reference/files]] */
+  def getFiles: Request[Either[ResponseException[String, Exception], FilesResponse]] =
+    openApiAuthRequest
+      .get(OpenAIEndpoints.FilesEndpoint)
+      .response(asJsonSnake[FilesResponse])
+
+  /** Upload a file that contains document(s) to be used across various endpoints/features. Currently, the size of all the files uploaded by
+    * one organization can be up to 1 GB. Please contact OpenAI if you need to increase the storage limit.
+    * @param file
+    *   [[java.io.File File]] of the JSON Lines file to be uploaded. <p> If the purpose is set to "fine-tune", each line is a JSON record
```

Comment:
[minor] what's the documentation benefit of `[[java.io.File File]]`? The parameter type is in the signature already, as well as the name is known?